packages/core/src/modules/basic-messages/BasicMessagesApi.ts

```diff
@@ -41,13 +41,14 @@ export class BasicMessagesApi {
    * @throws {MessageSendingError} If message is undeliverable
    * @returns the created record
    */
-  public async sendMessage(connectionId: string, message: string) {
+  public async sendMessage(connectionId: string, message: string, parentThreadId?: string) {
```

Comment:
Would be good to overhaul this API, but as you said, we can do that after 0.4.0 is released